CoffeeShop.playground/Sources/CoffeeShopAppTest.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+import XCTest
+
+public class CoffeeShopAppTest: XCTestCase {
+
+    public override func setUp() {
```

Comment:
@eandreea You should avoid overriding methods with no implementation. Is just "noise" to code base, no value added.